src/functionalTest/java/uk/gov/hmcts/reform/bulkscan/orchestrator/CreateCaseTest.java

```diff
@@ -0,0 +1,166 @@
+     */
+    private String invokeCallbackEndpoint(
+        CaseDetails exceptionRecord
+    ) throws IOException {
+        CaseDetails exceptionRecordWithSearchFields =
+            exceptionRecord.toBuilder().build();
+
+        CallbackRequest callbackRequest = CallbackRequest
+            .builder()
+            .eventId(EVENT_ID_CREATE_NEW_CASE)
+            .caseDetails(exceptionRecordWithSearchFields)
+            .build();
+
+        CcdAuthenticator ccdAuthenticator = ccdAuthenticatorFactory.createForJurisdiction("BULKSCAN");
+
+        Response response = RestAssured
+            .given()
+            .relaxedHTTPSValidation()
+            .baseUri(testUrl)
+            .header(HttpHeaders.CONTENT_TYPE, MediaType.APPLICATION_JSON_VALUE)
+            .header(SyntheticHeaders.SYNTHETIC_TEST_SOURCE, "Bulk Scan Orchestrator Functional test")
+            .header(AUTHORIZATION, ccdAuthenticator.getUserToken())
+            .header(CcdCallbackController.USER_ID, ccdAuthenticator.getUserDetails().getId())
+            .body(callbackRequest)
+            .when()
+            .post("/callback/create-new-case");
+
+        return getCaseCcdId(response);
+    }
+
+    private String getCaseCcdId(Response response) throws IOException {
+        assertThat(response.getStatusCode()).isEqualTo(200);
+
+        final AboutToStartOrSubmitCallbackResponse callbackResponse =
+            new ObjectMapper().readValue(response.getBody().asString(), AboutToStartOrSubmitCallbackResponse.class);
+        assertThat(callbackResponse.getData()).isNotNull();
+        assertThat(callbackResponse.getData().containsKey(CASE_REFERENCE)).isTrue();
+        return (String) callbackResponse.getData().get(CASE_REFERENCE);
+    }
+
+    private CaseDetails createExceptionRecord(String resourceName) throws Exception {
+        // TODO use envelopeId for search
+        UUID poBox = UUID.randomUUID();
+
+        envelopeMessager.sendMessageFromFile(resourceName, "0000000000000000", null, poBox, dmUrl);
+
+        await("Exception record is created")
+            .atMost(60, TimeUnit.SECONDS)
+            .pollDelay(2, TimeUnit.SECONDS)
+            .until(() -> lookUpExceptionRecord(poBox).isPresent());
+
+        CaseDetails caseDetails = lookUpExceptionRecord(poBox).get();
+        return caseDetails;
+    }
+
+    private Optional<CaseDetails> lookUpExceptionRecord(UUID poBox) {
+        List<CaseDetails> caseDetailsList = caseSearcher.search(
+            SampleData.JURSIDICTION,
+            SampleData.JURSIDICTION + "_" + CreateExceptionRecord.CASE_TYPE,
+            ImmutableMap.of(
+                "case.poBox", poBox.toString()
+            )
+        );
+        return caseDetailsList.stream().findFirst();
+    }
+
+    private Boolean isCaseCreated(String caseCcdId, CaseDetails exceptionRecord) {
+
+        CaseDetails createdCase = ccdApi.getCase(
+            String.valueOf(caseCcdId),
+            exceptionRecord.getJurisdiction()
```

Comment:
you could pass just the jurisdiction to this method

Reply:
Have fixed